src/sandbox.rkt

```diff
@@ -146,7 +145,9 @@
     (define est-end-score (errors-score (test-success-end-est-error result)))
 
     ;; TODO: this is broken because errors are always ordinary values now!
-    (define-values (reals infs) (partition ordinary-value? (map - end-errors start-errors)))
+    (define binary64 (get-representation 'binary64))
+    (define-values (reals infs) (partition (curryr ordinary-value? binary64)
+                                           (map - end-errors start-errors)))
```

Comment:
This code also doesn't do anything any more, but I guess let's clean it up as a separate PR.